containersQa/testlib.bash

```diff
@@ -545,6 +545,12 @@ function s2iLocal() {
     else
       cat $DF.orig | sed "s;/s2i/run;/s2i/run $ADDS;g;" > $DF.nw1
     fi
+    if [ "x$COMMIT_HASH" != x ] ; then
+      #cd into the source folder and checkout the koji source based on the known commit
+      pushd upload/src
+         git checkout $COMMIT_HASH
```

Comment:
I would recommend to quote around the $hash

Reply:
Good catch. I had to look up the why. It makes sense to keep that as a best practice. Thank you.